tests/Tests/400__ods_dashboard/414__ods_dashboard_api.robot

```diff
@@ -0,0 +1,374 @@
+
+Verify Access To console-links API Endpoint
+    [Documentation]     Verifies the endpoint "console-links" works as expected
+    ...                 based on the permissions of the user who query the endpoint
+
+    [Tags]    ODS-XYZ
+    ...       Tier1
+    ...       Security
+    Perform Dashboard API Endpoint GET Call   endpoint=${CONSOLE_LINKS_ENDPOINT}    token=${BASIC_USER_TOKEN}
+    Operation Should Be Allowed
+    Perform Dashboard API Endpoint GET Call   endpoint=${CONSOLE_LINKS_ENDPOINT}    token=${ADMIN_TOKEN}
+    Operation Should Be Allowed
+
+Verify Access To docs API Endpoint
+    [Documentation]     Verifies the endpoint "docs" works as expected
+    ...                 based on the permissions of the user who query the endpoint
+
+    [Tags]    ODS-XYZ
+    ...       Tier1
+    ...       Security
+    Perform Dashboard API Endpoint GET Call   endpoint=${DOCS_ENDPOINT}    token=${BASIC_USER_TOKEN}
+    Operation Should Be Allowed
+    Perform Dashboard API Endpoint GET Call   endpoint=${DOCS_ENDPOINT}    token=${ADMIN_TOKEN}
+    Operation Should Be Allowed
+
+Verify Access To getting-started API Endpoint
+    [Documentation]     Verifies the endpoint "getting_started" works as expected
+    ...                 based on the permissions of the user who query the endpoint
+
+    [Tags]    ODS-XYZ
+    ...       Tier1
+    ...       Security
+    Perform Dashboard API Endpoint GET Call   endpoint=${GETTING_STARTED_ENDPOINT}    token=${BASIC_USER_TOKEN}
+    Operation Should Be Allowed
+    Perform Dashboard API Endpoint GET Call   endpoint=${GETTING_STARTED_ENDPOINT}    token=${ADMIN_TOKEN}
+    Operation Should Be Allowed
+
+Verify Access To quickstarts API Endpoint
+    [Documentation]     Verifies the endpoint "quickstarts" works as expected
+    ...                 based on the permissions of the user who query the endpoint
+
+    [Tags]    ODS-XYZ
+    ...       Tier1
+    ...       Security
+    Perform Dashboard API Endpoint GET Call   endpoint=${QUICKSTARTS_ENDPOINT}    token=${BASIC_USER_TOKEN}
+    Operation Should Be Allowed
+    Perform Dashboard API Endpoint GET Call   endpoint=${QUICKSTARTS_ENDPOINT}    token=${ADMIN_TOKEN}
+    Operation Should Be Allowed
+
+Verify Access To segment-key API Endpoint
+    [Documentation]     Verifies the endpoint "segment-key" works as expected
+    ...                 based on the permissions of the user who query the endpoint
+
+    [Tags]    ODS-XYZ
+    ...       Tier1
+    ...       Security
+    Perform Dashboard API Endpoint GET Call   endpoint=${SEGMENT_KEY_ENDPOINT}    token=${BASIC_USER_TOKEN}
+    Operation Should Be Allowed
+    Perform Dashboard API Endpoint GET Call   endpoint=${SEGMENT_KEY_ENDPOINT}    token=${ADMIN_TOKEN}
+    Operation Should Be Allowed
+
+Verify Access To gpu API Endpoint
+    [Documentation]     Verifies the endpoint "segment-key" works as expected
+    ...                 based on the permissions of the user who query the endpoint
```

Comment:
endpoint should be "gpu"

Reply:
ops, copy-paste mistake xD thx I'll fix it